etc/kayobe/ansible/stackhpc-openstack-tests.yml

```
@@ -68,6 +68,7 @@
               --self-contained-html
               --pyargs stackhpc_cloud_tests
               --timeout {{ sot_timeout }}
+              -rfEx
```

I believe the x is redundant (and fE is the default:
>   -r chars              Show extra test summary info as specified by chars: (f)ailed, (E)rror, (s)kipped, (x)failed, (X)passed, (p)assed, (P)assed with output, (a)ll except passed (p/P), or (A)ll. (w)arnings are enabled by default (see --disable-warnings), 'N' can be used to reset the list.(default: 'fE').

```suggestion
              -r
```